log/handler_glog.go

```diff
@@ -198,11 +198,11 @@ func (h *GlogHandler) Log(r *Record) error {
 		}
 	}
 	// If the global log level allows, fast track logging
-	if atomic.LoadUint32(&h.level) >= uint32(r.Lvl) {
+	if h.level.Load() >= uint32(r.Lvl) {
 		return h.origin.Log(r)
 	}
 	// If no local overrides are present, fast track skipping
-	if atomic.LoadUint32(&h.override) == 0 {
+	if h.override.Load() {
```

Comment:
It needs to be either 
`h.override.Store(len(filter) != 0)` and check `if !h.override.Load() {`
or 
`h.override.Store(len(filter) == 0)` and check `if h.override.Load() {`